Dockerfile

```diff
@@ -16,12 +16,21 @@ RUN apt-get update -qq && apt-get install -y \
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 
 # Add Chrome source
-RUN curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
 
-RUN apt-get update -qq
-RUN apt-get install -y nodejs libnss3 libgconf-2-4 google-chrome-stable
-RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+# Temporary install older chrom version
+# Can be reverted back to stable once this project is under Ruby 3
+# and Selenium is updated
+
+ENV CHROME_VERSION 114.0.5735.90-1
+RUN curl -o /tmp/chrome-114.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb
+RUN apt-get update -qq && apt-get install -y nodejs libnss3 libgconf-2-4 && apt-get install -y /tmp/chrome-114.deb && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# RUN curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+# RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
+
+# RUN apt-get update -qq
+# RUN apt-get install -y nodejs libnss3 libgconf-2-4 google-chrome-stable
+# RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
**suggestion / non-blocking**: Up to you but we already have this change as a separate commit to revert later so I'd suggest removing the lines altogether instead of commenting them out.